test/hotspot/jtreg/serviceability/jvmti/vthread/SelfSuspendDisablerTest/libSelfSuspendDisablerTest.cpp

```diff
@@ -58,6 +58,13 @@ Java_SelfSuspendDisablerTest_resumeAllVirtualThreads(JNIEnv* jni, jclass cls) {
   check_jvmti_status(jni, jvmti->ResumeAllVirtualThreads(0, NULL), "Error in ResumeAllVirtualThreads");
 }
 
+JNIEXPORT jint JNICALL
+Java_SelfSuspendDisablerTest_getThreadState(JNIEnv* jni, jclass cls, jthread thread) {
+  jint state;
+  check_jvmti_status(jni, jvmti->GetThreadState(thread, &state), "Error in GetThreadState");
+  return state;
+}
+
 }
```

Comment:
it might be helpful to add 

// extern "C" 

after the brace as it confused me initially as to why there are two braces.

Reply:
Good catch. Fixed.

Reply:
Good suggestion. Resolved.